ext/configuration.c

```diff
@@ -156,6 +157,51 @@ static bool dd_parse_tags(zai_str value, zval *decoded_value, bool persistent) {
     return true;
 }
 
+static bool dd_parse_http_error_statuses(zai_str value, zval *decoded_value, bool persistent) {
```

Comment:
Is there anything the default parser for set does not handle (which essentially simply splits on colon), that we need a custom parser here?